libs/core/execution/tests/unit/executor_parameters.cpp

```diff
@@ -88,7 +88,13 @@ void test_static_chunk_size()
         parameters_test(scs);
     }
 }
-
+void test_adaptive_static_chunk_size()
```

Comment:
Is this function invoked anywhere?

Reply:
problem should be solved in the new push. Thanks!